osidb/migrations/0132_flawcomment_redesign.py

```diff
@@ -0,0 +1,48 @@
+from django.db import migrations, models
+from django.conf import settings
+
+from osidb.core import set_user_acls
+
+
+BATCH_SIZE = 1000
+
+
+def forwards_func(apps, schema_editor):
+    set_user_acls(settings.ALL_GROUPS)
+    FlawComment = apps.get_model("osidb", "FlawComment")
+
+    # Move existing comment data from meta_attr to the model
+    comments = FlawComment.objects.all().iterator(chunk_size=BATCH_SIZE)
+
+    batch = []
+    for i, comment in enumerate(comments, 1):
+        comment.creator = comment.meta_attr.get("creator", "")
+        # is_private seems to be stored as a string in old data
```

Comment:
Note: This is not entirely true. `meta_attr` is hstore field type and such field can only hold key:values records where each value is a string, meta_attr simply cannot hold any other type. I believe in old data, there is a boolean in place.

Reply:
I see. I will check for both types then just in case. Inconsistencies like this are yet another good reason to make the switch.